web_widget_uom/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module aims to allow the user to decide how many
+decimal places should be displayed in the Odoo User
+Interface for each Unit of Measure.
```

Comment:
It should mention that `decimal.precision.digits` must be greater than uom.uom.decimal_places. Because if this is not the case it will be truncated nevertheless, or you can check it in Python.
A check in python is better IMO.

Reply:
@tarteo i've pushed some changes. 

Basically it now show min decimal places between decimal places set and decimal uom precision. Could u check?